contrib/IECoreUSD/src/IECoreUSD/DataAlgo.cpp

```diff
@@ -142,9 +159,10 @@ IECore::DataPtr dataFromArray( const pxr::VtValue &value, GeometricData::Interpr
 
 IECore::DataPtr dataFromSdfAssetPath( const SdfAssetPath &assetPath, const pxr::UsdAttribute *attribute = nullptr )
 {
-	if( assetPath.GetResolvedPath().size() || !assetPath.GetAssetPath().size() || !attribute )
+	const std::string p = DataAlgo::fromUSD( assetPath );
+	if( p.size() || !assetPath.GetAssetPath().size() || !attribute )
 	{
-		return new StringData( assetPath.GetResolvedPath() );
+		return new StringData( p );
 	}
```

Comment:
There's an additional return path from this function at L189 that I think should get the same treatment.

Reply:
Fixed in https://github.com/ImageEngine/cortex/pull/1426/commits/ae0f1ba29e3a2c147006bbceca67b5a44f31fb67